src/MSALWrapper/AuthFlow/AuthFlowExecutor.cs

```diff
@@ -17,15 +19,18 @@ public class AuthFlowExecutor : IAuthFlow
     {
         private readonly IEnumerable<IAuthFlow> authflows;
         private readonly ILogger logger;
+        private readonly ITelemetryService telemetryService;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="AuthFlowExecutor"/> class.
         /// </summary>
         /// <param name="logger">The logger.</param>
+        /// <param name="telemetryService">The telemetry service.</param>
         /// <param name="authFlows">The list of auth flows.</param>
-        public AuthFlowExecutor(ILogger logger, IEnumerable<IAuthFlow> authFlows)
+        public AuthFlowExecutor(ILogger logger, ITelemetryService telemetryService, IEnumerable<IAuthFlow> authFlows)
         {
             this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            this.telemetryService = telemetryService;

```

Comment:
Consider following the same pattern here that we expect non-null args and throw an exception if given a null arg.

Reply:
+1 to Kyle.

Reply:
Got it!